runtime/src/main/java/org/corfudb/runtime/clients/ClientMsgHandler.java

```diff
@@ -77,8 +77,8 @@ public boolean handle(CorfuMsg message, ChannelHandlerContext ctx) {
                 if (ret != null) {
                     client.getRouter().completeRequest(message.getRequestID(), ret);
                 }
-            } catch (Exception ex) {
-                client.getRouter().completeExceptionally(message.getRequestID(), ex);
+            } catch (Exception | Error e) {
```

Comment:
do you want all Throwables or this is wide enough.

Reply:
Error and Exception share the same parent type (i.e. Throwable). Changed it to that. 
Errors are thrown by reflection methods, so we need to catch them here. Later on with the RPC changes, we can get rid of it.